components/molecule/accordion/src/index.scss

```diff
@@ -99,6 +100,7 @@
     background-color: $bgc-accordion-tab-container;
     position: relative;
     width: 100%;
+    border-radius: $bdrs-accordion-tab;
```

Comment:
🍆 if it's no different than the border radius for the other children, do you really need this property here (using the same `$bdrs-accordion-tab` variable?

Reply:
Yes, it needs to be applied on both selectors.